lib/msal-browser/test/app/PublicClientApplication.spec.ts

```diff
@@ -1081,13 +1085,7 @@ describe("PublicClientApplication.ts Class Unit Tests", () => {
         });
 
         it("Emits acquireToken success event if user was already signed in", async () => {
-            const testAccount: AccountInfo = {
-                homeAccountId: TEST_DATA_CLIENT_INFO.TEST_HOME_ACCOUNT_ID,
-                localAccountId: TEST_DATA_CLIENT_INFO.TEST_UID,
-                environment: "login.windows.net",
-                tenantId: "3338040d-6c67-4c5b-b112-36a304b66dad",
-                username: "AbeLi@microsoft.com",
-            };
+            const testAccount = BASIC_TEST_ACCOUNT_INFO;
```

Comment:
This is a great bit of refactoring/cleanup here.  Thanks!

Reply:
Account Info should be generated from a proper test ID token basically.